### ISSUE TITLE
Wrap Math::getPPM

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#5772]
currently a template

**Comment by tapaswenipathak** [comment]
can this have more information?

**Comment by timosachsenberg** [comment]
It is not wrapped in python e.g. the method is missing from the cython pxd file because it is a template. @jpfeuffer maybe something for Erik

**Comment by jpfeuffer** [comment]
Yes but templates are usually not a problem. So someone just forgot or didn't know how to do it, right?

**Comment by FreezB11** [comment]
sir i checked the files itseems it is wrapped now
ig we can close this issue

**Comment by jpfeuffer** [comment]
Link? Where is it wrapped in python?

**Comment by FreezB11** [comment]
Aaah I am really sorry sir I saw the code in c++ and got confused 


umm sir then do i have to make the .pxd file for it.

**Comment by SACHIN2026** [comment]
@tapaswenipathak @timosachsenberg @jpfeuffer @FreezB11 @pjones  Can I work on this issue
